modules/avatar/react/lib/Avatar.tsx

```diff
@@ -6,16 +6,16 @@ import {focusRing, hideMouseFocus} from '@workday/canvas-kit-react-common';
 import {SystemIconCircle, SystemIconCircleSize} from '@workday/canvas-kit-react-icon';
 import {userIcon} from '@workday/canvas-system-icons-web';
 
-export enum AvatarTheme {
+export enum Variant {
```

Comment:
Should we name this enum `AvatarVariant`?

Reply:
Yes we should 👍

Reply:
Done!